pkg/database/fake/database_generated_test.go

```diff
@@ -145,6 +145,86 @@ func TestQueryNamespacedResources(t *testing.T) {
 	}
 }
 
+func TestQueryNamespacedResourceByName(t *testing.T) {
```

Comment:
I miss the "more than one found" error.

Reply:
Right, and I missed adding tests over the actual database logic. I've just updated the PR to add them.